codec/fuzz/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "commonware-codec-fuzz"
```

Comment:
Can you provide the rationale for using an inner crate here rather than just implementing fuzzing on `commonware-codec` (like we do for benches)? I'm assuming there is a good rationale I just don't know about.